app/(main)/league/[leagueId]/entry/all/page.tsx

```diff
@@ -87,13 +86,17 @@ const Entry = ({
     <>
       {entries.map((entry) => {
         const linkUrl = `/${LEAGUE_URL}/${leagueId}/${ENTRY_URL}/${entry.$id}/${WEEK_URL}/${currentWeek}`;
+        const isPickSet = entry.selectedTeams.length > 0;
+        const teamLogo = isPickSet ? entry.selectedTeams[0].teamLogo : '';
 
         return (
           <section key={entry.$id}>
             <LeagueEntries
               key={entry.$id}
               entryName={entry.name}
+              isPickSet
               linkUrl={linkUrl}
+              teamLogo={teamLogo}
```

Comment:
If you're going to change the shorthand for isPickSet, you mind as well do it for teamLogo too.

Reply:
When I tried to shorthand it, the variable `teamLogo` in line 90 stated that this variable has been declared and never used so I'd like to keep this together. Thoughts?

Reply:
That's an ES Lint rule. It probably cannot read shorthand variables.

Reply:
Ah, got it. Thanks for letting me know!